### ISSUE TITLE
new to coding having problems trying to make an NFT collection

### ISSUE THREAD

**Issue by Tequilako** [HashLips/hashlips_art_engine#520]
I try to follow every step of this video
https://www.youtube.com/watch?app=desktop&v=AaCgydeMu64&t=1096
In the video 24:40-33:00 
I did everything right, but the code "node utils/nftport/uploadFiles.js" is still not working.
Can somebody help me please? I appreciated. 

<img width="796" alt="Screen Shot 2022-01-13 at 9 22 23 PM" src="https://user-images.githubusercontent.com/97678728/149337931-eb4590e0-8838-4bec-9566-dcf42a48779e.png">

**Comment by KaiWenF** [comment]
I am having a problem as well with the uploadFiles.js!   I too watched the video several times to see if I missed something but I can't tell that I have.  The only thing that I have changed in the code is adding my AUTH key and that is it.  Have not been able to figure out the problem and I have not gotten any feed back from anyone.  Has your issue been resolved?

**Comment by Tequilako** [comment]
Unfortunately, not yet.

**Comment by KaiWenF** [comment]
Oh man!  I might just have to find another solution.  Usually I can figure
out most coding issues but this one has me beat.  Time for me to move on to
different method of getting my NFT art uploaded.

**Comment by thepeanutgalleryandco** [comment]
@Tequilako @KaiWenF you are more than welcome to try out my repo that is based on this repo and codestacr's repo and now also has erc1155 support. There is a big README doc with examples at the bottom of how to follow all the steps.

https://github.com/thepeanutgalleryandco/create-and-mint-nft-collection

**Comment by KaiWenF** [comment]
Thanks Rohan,

Thank you for replying to my message.  I will give your code a try to see
what happens.  So your uploadFiles.js file is working for you?  I will let
you know how it goes.

Regards,
Kai

**Comment by KaiWenF** [comment]
How my goodness I got a copy of your code and I am reviewing it now.  Wow
you really made some changes.  I am about to run the code to mint my NFTs
now after I do the layer setup.  If everything works and I can run the code
in the nftport file I will GLADLY send you a little something for your
work.  :)

Regards,
Kai

**Comment by thepeanutgalleryandco** [comment]
Yeah, I released my update on Friday afternoon and tested both the ERC721 no reveal example as well as the ERC1151 no reveal example without any issues. Let me know how it goes or ping me if you need any help. I think Hashlips and codeSTACKr is swamped with everyone trying the code bases and different cases for their collections. Unfortunately I am not that good at video making etc., so I dont have a video out to guide people. Hopefully the massive README doc is enough!

Good luck 😎

**Comment by KaiWenF** [comment]
Hi Rohan,

I am reading through your documentation on your GIT as well as the code
comments.  They are super helpful and I have been able to get 5 test images
up on NFTport.  I've not tried the mint process yet.  I was up super late
until about 2am in the morning studying your code and the documentation.  I
am going to perform the minting process on my 5 images I placed in rinkeby
to go all the way to the Opensea site to see the images.  I just now
finished applying to NFTport for the "Community" account that will allow me
to upload 10k of my NFT items.  I am baffled as to the problem that was
happening in codeSTACKr "uploadFiles.js file.  I worked on that issue I was
having for about a week and Hashlips told me that he couldn't help with the
issue since it was some code that was added to his original project, which
I totally understood.  As a developer I am curious as to how you got around
the problem(s) with the "uploadFiles.js" code.  I am a .net developer
coding in vb.net and c#, some javascript but my javascript is mainly for
client side processes for my asp.net pages.  I would like to think I am a
pretty competent software developer but I was hitting a brick wall.  Yes
the README is AWESOME and since I am a developer I can follow your logic.
The NFT world is very new to me but it looks so interesting and I look
forward to sharing my art.  Please send me paypal if you have one so that I
can send you a little something for all of your effort.  I hope that is
cool.  I will have to add you to my budget for the month of January.

If I run into any issues I will share with you, but so far everything is
pretty solid.

Best Regards,
Kai

**Comment by thepeanutgalleryandco** [comment]
Howzit Kai,

I am so amped for you that you finally got going and that you are persevering in the journey of NFTs even though you have hit a few minor roadblocks.

I am a Java and PL/SQL dev and have never worked in Javascript before. I actually only dev'ed a few months as I work mostly on the solution and configuration side in my day job. I am mostly googling how to do X and Y in Javascript and learning as I am going and the repo is teaching me some nice tricks! 

I am not sure what the latest issues are on the parent repos, I will need to download them again and run through it to see the issues and see how best to solve them. When I started on my 10k collection, I ran into multiple issues and had to google how to fix them and built scripts for myself. I soon noticed a lot of people in the community was struggling as well so I decided to start my own repo on the epic code base that Hashlips and codeSTACKr already built. I fixed the issues mainly by re-writing the code for the most parts into how it made sense to me, but still kept the framework to work on the parent code base's file structures. This is actually V2 of the code base, as V1 used different base structures and it was too difficult for people to migrate to my code base if they started on one of the others. It was also difficult to pull in changes kn the parent code bases into my code base.

Hope that it was of help and you honestly do not need to give me anything, I am just happy that you stuck with it and that you are getting it over the line. If you feel you really want to send me something, then you are welcome to pick up a few NFTs from my collection, send something to my MEW address or if you want to do paypal, then you can use rohandejongh@yahoo.com.

Cheers bud, looking forward to seeing your collection on Opensea!

**Comment by KaiWenF** [comment]
Hey Rohan,

I WILL BE SENDING YOU A little something soon.  I am on this once a month
pay cycle so I will let you know.  I am hoping to put up 10,000 of my NFT
items soom.  I got the test completed today on Opensea Testnets and
everything looks good.  So since you have 10K of your NFTs up how did you
go about selling them and setting a price for each of them?  I have my work
cut out for me!    Some of the NFT items are "rare" and "super rare".   I
am going to do a little twitter marketing and get my kaiwen.com website set
up to show my NFTs.  I saw a cool WordPress add-on I want to use for
showing my NFTs.  I am still waiting to hear back from NFTPort to see if I
can upload 10k items.   I will also buy one of your NFT items too.  It will
be my first.  I am still looking at your code and studying it.  JavaScript
is like a close cousin to C so the logic is not hard to follow.  I was
wondering if you have tried to  "pixelate" 10k items?  I tried and ran out
of memory.  It could have something to do with me running on a virtual
instance of Windows 10 using Parrallels!  ha ha ha!  I am running visual
studio 2022 which is pretty cool.  I love the intellisense!!  Since I have
everything working good I'm going to check in my code to GIT and then back
up the Parrallel file to another harddrive, just in case!!

THANK YOU,

Kai

**Comment by thepeanutgalleryandco** [comment]
Hi Kai,

Awesome news that you got them uploaded and working, even if it is testnet 🎊🎉🎊🎉

Unfortunately, the only way to unhide items from your profile on Opensea currently is a manual process and you can unhide 20 at a time. The sale process is even worse at only one item at a time. Some people are able to setup a macro process to do this, however, I am on a mac and I am yet to find a solution for the macro recorder. Opensea also has problems at times with displaying / refreshing the data, so I took the manual root of unhiding all the items.

If you do not get into community for NFTPort, you will need to get the growth plan and then maybe do 500 NFTs per month, depending on what you can afford ofcourse. I have seen so many cases where people uploaded 10k collections, but then struggle to sell 2 NFTs. It might be better to have a 10k collection, but slowly upload and release them instead of having the full collection uploaded.

Good luck 😎

**Comment by Merc616** [comment]
All you have to do is go to terminal and type npm i node-fetch@2.6.1
